src/XCCDF_POLICY/xccdf_policy.c

```diff
@@ -981,6 +979,52 @@ _xccdf_policy_rule_evaluate(struct xccdf_policy * policy, const struct xccdf_rul
 		return _xccdf_policy_report_rule_result(policy, result, rule, NULL, XCCDF_RESULT_NOT_SELECTED, NULL);
 	}
 
+	bool all_requires_met = true;
```

Comment:
I don't understand how this should be done at resolve stage. 
I didn't manage to reach `xccdf_resolve_rule` or `xccdf_resolve_group` during evaluation.

If I understand correctly, by looking at XCCDF specification, resolving is related to loading the XCCDF document.
And the `requires` and `conflicts` are handled during traversal, and described in section "Item Processing Algorithm".